src/groupeddataframe/groupeddataframe.jl

```diff
@@ -267,7 +267,7 @@ function Base.NamedTuple(key::GroupKey)
     N(_groupvalues(parent(key), getfield(key, :idx)))
 end
 Base.Tuple(key::GroupKey) = values(key)
-
+Base.Dict(key::GroupKey) = Dict(String(first(ki)) => last(ki) for ki in pairs(key))
```

Comment:
This raises again the question of whether we should use symbols or strings by default... Another place where we had to choose is `eachcol`/`DataFrameColumns`. We should have a general policy about this.

Reply:
The general policy is to use `Symbol` in this case (which was settled after I looked at this PR).
I know @pdeffebach voted otherwise :smile:, but we should be consistent.

Reply:
Ah - as @nalimilan prompted me - we have not settled the decision in https://github.com/JuliaData/DataFrames.jl/issues/2298#issuecomment-646703922. So first we should decide there and then update this PR accordingly.